HOW/info/README.md

```diff
@@ -0,0 +1,2 @@
+# Informational Documents
+Information about Holochain design issues, or general guidelines or information to the Holochain community.  These documents aren't technical or social standards, but are useful information that has been aligned on through the HOW process.
```

Comment:
I can't fathom what this might be. Maybe add an example? Or remove this info-type until a real need for it arises?

Reply:
So for me this would be a place where we might put explanatory texts that define terms that are being used in the context, exactly like what we were talking about in our meeting around Privacy/Security/Secrecy. 

I'm not wed to this, could remove for the sake of simplicity.

Reply:
I'm good either way, just need an example in the text if you'd like to keep it : )

Reply:
Same for me, without an example or more specificity this is too abstract for me to grasp.